pkg/status/render/templates/aggregator.tmpl

```diff
@@ -5,53 +5,55 @@ NOTE: Changes made to this template should be reflected on the following templat
 ==========
 Aggregator
 ==========
-{{- if .ChecksMetricSample }}
-  Checks Metric Sample: {{humanize .ChecksMetricSample}}
-{{- end }}
-{{- if .DogstatsdMetricSample}}
-  Dogstatsd Metric Sample: {{humanize .DogstatsdMetricSample}}
-{{- end }}
-{{- if .Event}}
-  Event: {{humanize .Event}}
-{{- end }}
-{{- if .EventsFlushed}}
-  Events Flushed: {{humanize .EventsFlushed}}
-{{- end }}
-{{- if .EventsFlushErrors}}
-  Events Flush Errors: {{humanize .EventsFlushErrors}}
-{{- end }}
-{{- if .NumberOfFlush}}
-  Number Of Flushes: {{humanize .NumberOfFlush}}
-{{- end }}
-{{- if .SeriesFlushed}}
-  Series Flushed: {{humanize .SeriesFlushed}}
-{{- end }}
-{{- if .SeriesFlushErrors}}
-  Series Flush Errors: {{humanize .SeriesFlushErrors}}
-{{- end }}
-{{- if .ServiceCheck}}
-  Service Check: {{humanize .ServiceCheck}}
-{{- end }}
-{{- if .ServiceCheckFlushed}}
-  Service Checks Flushed: {{humanize .ServiceCheckFlushed}}
-{{- end }}
-{{- if .ServiceCheckFlushErrors}}
-  Service Checks Flush Errors: {{humanize .ServiceCheckFlushErrors}}
-{{- end }}
-{{- if .SketchesFlushed}}
-  Sketches Flushed: {{humanize .SketchesFlushed}}
-{{- end }}
-{{- if .SketchesFlushErrors}}
-  Sketches Flush Errors: {{humanize .SketchesFlushErrors}}
-{{- end }}
-{{- if .ChecksHistogramBucketMetricSample }}
-  Checks Histogram Bucket Metric Sample: {{humanize .ChecksHistogramBucketMetricSample}}
-{{- end }}
-{{- if .EventPlatformEvents }}
-{{- range $k, $v := .EventPlatformEvents }}
-  {{ $k }}: {{humanize $v}}
-{{- end }}
-{{- end }}
-{{- if .HostnameUpdate}}
-  Hostname Update: {{humanize .HostnameUpdate}}
+{{- with .aggregatorStats }}
```

Comment:
Should this be before the header above ?

Reply:
Same as with the other question. Looks like we were always rendering the aggregator headers even if there were no `aggregatorStats`